CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 
 ### Internal changes ⚙️
 
+* `QiskitDevice` does not issue a warning when device shots are `None` shots at initialization any more.
+  Instead, a `no_analytic` preprocess transform is applied.
+  [(#650)](https://github.com/PennyLaneAI/pennylane-qiskit/pull/650)
```

Comment:
I'm sure Isaac will have thoughts on the exact messaging, but I would think this should also be under `Breaking changes`, and should tell people how to fix their problem using the new and approved `set_shots` method. Something like:

```suggestion
* `QiskitDevice` does not issue a warning when device shots are `None` shots at initialization any more.
  Instead, a `no_analytic` preprocess transform is applied. This means an 
  error will now be raised if shots are not specified. Please use... 
  
[[ insert example with `set shots` here ]]
 
 on all `qnodes` that will be executed on the `QiskitDevice`.
  [(#650)](https://github.com/PennyLaneAI/pennylane-qiskit/pull/650)
```